src/models/searchservice/response/directanswer/FeaturedSnippetDirectAnswer.ts

```diff
@@ -19,7 +20,7 @@ export interface FeaturedSnippetDirectAnswer<T = unknown> extends DirectAnswer<T
   /** {@inheritDoc DirectAnswer.verticalKey} */
   verticalKey: string,
   /** {@inheritDoc DirectAnswer.fieldType} */
-  fieldType: BuiltInFieldType | string,
+  fieldType: EnumOrLiteral<BuiltInFieldType> | 'unknown',
```

Comment:
Can FeaturedSnippetDirectAnswers also have different field types and values? I would have thought a snippet is always a string?

Reply:
@tmeyer2115 talked to product, which they believe feature snippet can have other types

Reply:
Could we get a concrete example? I feel like we've tested FeaturedSnippetDirectAnswers before, but maybe not for non string types. It would be good for us to know how to do this in general, and if product doesn't know how to do this then that's probably a problem

Reply:
Arguably the fact that I don't know how to do it is a problem

Reply:
I think the rich text is a featured snippet. Here's an example
`"directAnswer":{
   "type":"FEATURED_SNIPPET",
   "answer":{
      "snippet":{
         "value":"# how to get rich text\n\ndrinks some coffee",
         "matchedSubstrings":[]
      },
      "fieldType":"rich_text"
   }
...`

Reply:
that did not format the way i wanted but yeah it says type is FEATURED_SNIPPET